src/pages/billing/Project.js

```diff
@@ -354,7 +354,7 @@ const ProjectDetail = ({ project, billingAccounts, authorizeAndLoadAccounts }) =
   const { displayName = null } = _.find({ accountName: billingProject.billingAccount }, billingAccounts) || { displayName: 'No Access' }
 
   return h(Fragment, [
-    billingAccountsOutOfDate && div({}, [h(BillingAccountSummaryPanel, { counts: _.mapValues(_.size, groups) })]),
+    billingAccountsOutOfDate && h(BillingAccountSummaryPanel, { counts: _.mapValues(_.size, groups) }),
```

Comment:
Thanks for fixing the styling here.

Reply:
Instead of changing the icon, move this to the end of the Fragment, which will cause the z-index to fix itself.

Reply:
@zarsky-broad Is there not merit to changing the icon? Someone else could encounter this with some of our icons and will have to remember that the additional transforms cause this?

Reply:
In this case, where the fix is to use the standard pattern of putting modals last in the render, I'm actually ok with it causing issues; we should assume that transforms could be used by other components.

Reply:
> the fix is to use the standard pattern 
Thanks - I had ordered them as I'd see them on the page, not according to modallity.